bug-bounty.md

```diff
@@ -0,0 +1,46 @@
+# Bug Bounty
+
+The Moonbeam bug bounty program is focused on the Moonriver and Moonbeam Parachains (deployed to Kusama and Polkadot respectively) and dAapps. It is focused on preventing:
+
+Moonbeam/Moonriver:
+* Thefts and freezing of principal of any amount
+* Thefts and freezing of unclaimed yield of any amount
+* Theft of governance funds 
+* Governance activity disruption
+* Network shutdown
```

Comment:
This could perhaps be more specific, maybe "block production interruption" or "chain stall" etc.

Reply:
I think it is ok to keep it vague, there are too many cases to describe otherwise